packages/sit-onyx/src/components/OnyxAccordionItem/OnyxAccordionItem.vue

```diff
@@ -0,0 +1,168 @@
+    :class="[densityClass]"
+    :open="isOpen"
+    @toggle="isOpen = ($event.target as HTMLDetailsElement).open"
+  >
+    <summary
+      :id="'header-' + itemId"
+      class="onyx-accordion-item__header"
+      role="button"
+      :tabindex="isDiabled ? -1 : 0"
+      :aria-expanded="isOpen"
+      :aria-controls="'panel-' + itemId"
+      :aria-disabled="isDiabled"
+    >
+      <div class="onyx-accordion-item__header-content">
+        <slot name="header"></slot>
+      </div>
+
+      <OnyxIcon :icon="chevronLeftSmall" class="onyx-accordion-item__header-icon" />
+    </summary>
+    <div
+      :id="'panel-' + itemId"
+      class="onyx-accordion-item__panel"
+      role="region"
+      :aria-labelledby="'header-' + itemId"
+    >
+      <slot></slot>
+    </div>
+  </details>
+</template>
+
+<style lang="scss">
+@use "../../styles/mixins/layers";
+.onyx-accordion-item {
+  @include layers.component() {
+    border-bottom: var(--onyx-1px-in-rem) solid var(--onyx-color-component-border-neutral);
+    color: var(--onyx-color-text-icons-neutral-intense);
+    font-family: var(--onyx-font-family);
+    position: relative;
+    width: 100%;
+
+    &__header {
+      width: 100%;
+      position: relative;
+      display: flex;
+
+      justify-content: space-between;
+      gap: var(--onyx-density-md);
+      align-items: center;
+      padding: var(--onyx-density-md);
+
+      cursor: pointer;
+      &:hover,
+      &:focus-visible {
+        background-color: var(--onyx-color-base-neutral-200);
+        outline: none;
+      }
+      &-content {
+        display: flex;
+        * {
+          font-size: 1rem;
+          font-weight: 400;
+        }
+      }
+
+      &-icon {
+        color: inherit;
+      }
+    }
+    &[open] &__header-icon {
+      transform: rotate(-90deg);
+    }
+    &:has(:focus-visible) {
+      border-radius: var(--onyx-radius-md);
+      border-bottom: none;
+      z-index: 1;
+      outline: var(--onyx-outline-width) solid var(--onyx-color-component-focus-primary);
+    }
+
+    &:not([open]) &__header {
+      color: var(--onyx-color-text-icons-neutral-medium);
+      &:hover,
+      &:focus-visible {
+        color: var(--onyx-color-text-icons-neutral-intense);
+      }
+    }
+    &[open] &__header::after {
+      transform: rotate(90deg);
+    }
+    &__panel {
+      padding: var(--onyx-density-md);
+    }
+    &:has(&__header[aria-disabled="true"]) {
+      color: var(--onyx-color-text-icons-neutral-soft);
+      pointer-events: none;
+      .onyx-accordion-item__header {
+        color: var(--onyx-color-text-icons-neutral-soft);
+      }
+    }
+  }
+  &-skeleton {
+    padding: var(--onyx-density-md);
+    display: flex;
+    gap: var(--onyx-density-md);
+    border-bottom: 1px solid var(--onyx-color-component-border-neutral);
+
+    &__main {
+      width: calc(100% - 3 * var(--onyx-density-md));
+      height: var(--onyx-density-md);
+    }
+    &__icon {
+      height: var(--onyx-density-md);
```

Comment:
If I see it correctly in Figma, the icon always has the same size for all densities. Only the padding changes

Reply:
Same for the main element height